accelerometer/LinearAccelerationSensor-threshold-manual.https.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
>LinearAccelerationSensor-threshold-manual.https.html

The file name looks like inappropriate, how about `LinearAccelerationSensor-shake-threshold-manual.https.html`?

Reply:
Updated, @Honry, PTAL.